.github/workflows/build.yml

```
@@ -71,22 +74,26 @@
+          fetch-depth: 2
+
       - name: Install pnpm
         uses: pnpm/action-setup@v4
         with:
           version: 9
+
       - name: Get pnpm store directory
         shell: bash
         run: |
           echo "STORE_PATH=$(pnpm store path --silent)" >> $GITHUB_ENV
+          
```

yamllint

[error] trailing spaces

(trailing-spaces)